### DISCUSSION TITLE
Cell capacity definition

### DISCUSSION THREAD

**Discussion by jouslin** [pybamm-team/PyBaMM#4211] [Q&A] [answered]
Hi,

I've changed the cell capacity from 5 Ah (Chen 2020) to 10.7 Ah. To do this, I modified the parameter 'Nominal Cell Capacity [Ah]'. However, during a discharge experiment, while the current calculated from the nominal cell capacity appears correct, the discharge time does not reflect the new capacity. It seems that adjusting the nominal cell capacity only affects the current value, not the actual cell capacity itself.

Do you know how I can update the cell capacity?

Thank you.

**Comment by rtimms** [comment, marked as answer]
The parameter "Nominal Cell Capacity [A.h]" is just a nominal value used to convert between current and C-rate. To change the actual capacity you need to change some of the underlying physical parameters that affect capacity. The simplest way to do this is to change "Electrode height [m]" and "Electrode width [m]", the product of which sets the electrode cross-sectional area and therefore the capacity. You could also change things like active material volume fractions, maximum concentrations etc., depending on what you are trying to achieve.

**Reply by jouslin** [reply]
Thanks you very much for your help :)

**Comment by MarcoZangrandi** [comment]
Hi, is there a formula somewhere for calculating the nominal capacity? Because now I am having the same problem. To simulate a cylindrical LFP cell together with the thermal model, I had to take the Prada2013 set and the missing data I entered from Chen2020 which is the most complete set. However, Prada's nominal capacity is 2.3 Ah. By varying the parameters, however, I noticed that the actual rated capacity (discharge capacity after 1 hour at 1C) is not 2.3 but less. Thanks

**Reply by rtimms** [reply]
It is just a nominal value used to convert C rate to current. It doesn't actually affect the capacity of the cell.

**Reply by anmolpan** [reply]
Which parameter reflects the actual capacity of the cell?

**Reply by pkalbhor** [reply]
Actual capacity is defined by `Q_Li = y_100 * Q_p + x_100 * Q_n` which is defined [here](https://github.com/pybamm-team/PyBaMM/blob/977dcf9cdfc2fc978da3b1b56b7423454a2a2929/src/pybamm/models/full_battery_models/lithium_ion/electrode_soh.py#L136C17-L136C49).
Depends on `Q_n` and `Q_p` which are maximum capacities of respective electrodes. Their calculation is [here](https://github.com/pybamm-team/PyBaMM/blob/977dcf9cdfc2fc978da3b1b56b7423454a2a2929/src/pybamm/models/full_battery_models/lithium_ion/electrode_soh.py#L739-L740)
Here is the formula of `Q_n` for your convenience.
![image](https://github.com/user-attachments/assets/71b13396-6d9c-423c-8763-1b0bdc2f26e1)
More readable presentation would be 
![image](https://github.com/user-attachments/assets/24160be7-0657-4a82-8288-f5ee3c2b5118)

Hope it helps

**Reply by anmolpan** [reply]
Thank you!
